src/pages/api/auth/[...nextauth].ts

```diff
@@ -1,14 +1,25 @@
+import { createPersistedAuthAdapter } from "@/backend/auth/adapter";
+import { createContext } from "backend/context";
+import { NextApiHandler } from "next";
 import NextAuth from "next-auth";
 import GoogleProvider from "next-auth/providers/google";
 
-export default NextAuth({
-  // Configure one or more authentication providers
-  providers: [
-    GoogleProvider({
-      clientId: process.env.GOOGLE_CLIENT_ID as string,
-      clientSecret: process.env.GOOGLE_CLIENT_SECRET as string,
-    }),
+const handler: NextApiHandler = (req, res) => {
+  const { db } = createContext({
+    req,
+    res,
+  });
 
-    // ...add more providers here
-  ],
-});
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
Interesting - I don't get any error on this line for `npm run lint`

Reply:
![Screenshot 2023-04-13 at 9 57 21 PM](https://user-images.githubusercontent.com/7410405/231945499-ecc188c1-b8f9-4c4c-988a-4f68db8c857d.png)

hmm, did you `npm i`?

the error makes sense to me since the function is typed as any:

![Screenshot 2023-04-13 at 9 58 08 PM](https://user-images.githubusercontent.com/7410405/231945562-e4202ecf-2ffc-4159-867c-9b3138d951ad.png)